lovebank_services/routes.py

```diff
@@ -0,0 +1,97 @@
+from flask import jsonify, abort, request, make_response, url_for
+from lovebank_services import app    # import app from __init__.py
+from lovebank_services.models import User, Task
+
+tasks = [
+    {
+        'id': 1,
+        'title': 'Do the dishes',
+        'description': 'rinse and dry all the dishes in the kitchen sink',
+        'cost': 15,
+        'done': False
+    },
+    {
+        'id': 2,
+        'title': 'Do the laundry',
+        'description': 'Wash and dry all the dirty clothes',
+        'cost': 10,
+        'done': False
+    }
+]
+
+@app.route("/", methods=['GET'])
+def hello():
+    return "Hello, World!"
+
+
+def make_public_task(task):
+    new_task = {}
+    for field in task:
+        if field == 'id':
+            new_task['uri'] = url_for('get_task', task_id=task['id'],
+                                      _external=True)
+        else:
+            new_task[field] = task[field]
+    return new_task
+
+
+@app.route('/tasks', methods=['GET'])
+def get_tasks():
+    return jsonify({'tasks': [make_public_task(task) for task in tasks]})
+
+
+@app.route('/tasks/<int:task_id>', methods=['GET'])
+def get_task(task_id):
+    task = [task for task in tasks if task['id'] == task_id]
+    if len(task) == 0:
+        abort(404)
+    return jsonify({'task': make_public_task(task[0])})
+
+
+@app.route('/tasks', methods=['POST'])
+def create_task():
+    if not request.json or 'title' not in request.json:
+        abort(400)
+    task = {
+        'id': tasks[-1]['id'] + 1 if len(tasks) > 0 else 1,
+        'title': request.json['title'],
+        'description': request.json.get('description', ""),
+        'cost': request.json['cost'],
+        'done': False
+    }
+    tasks.append(task)
+    return jsonify({'task': make_public_task(task)}), 201
+
+
+@app.route('/tasks/<int:task_id>', methods=['PUT'])
```

Comment:
This code bombs for me.  You didn't import the six module.  When I import it, the code works. Can you check on this?

Reply:
Just added it